lms/templates/certificates/_accomplishment-related.html

```diff
@@ -0,0 +1,49 @@
+<div class="wrapper-accomplishment-related">
```

Comment:
If we'd like to leave the partial but just not reference it in the main view that might be ok. (my 0.2 cents)